api/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const cors = require("cors")
+const server = express();
+
```

Comment:
Nice and clean server file Stefan 👍 , it's easy to see what's going on and locate what Routers/resources your server is using 🔥